### DISCUSSION TITLE
Suggestions for adding a new mode in multiplayer

### DISCUSSION THREAD

**Discussion by Koinonia-1** [ppy/osu#25994] [Ideas] [answered]
**Please understand that meaning delivery may not be neat using a translator.**

In the current multi-play, everyone plays the same difficulty.

This is a perfect way from a competitive point of view, but it acts as a hurdle from a social point of view.

If the head of the multi-play room chooses a song so that people in a wide rank can enjoy multi-play together, I would like you to add a mode that each person can set the difficulty level.

**Comment by peppy** [comment]
We hope to support this sooner rather than later!

**Reply by Joehuu** [reply, marked as answer]
OP closed this, but it's already tracked in https://github.com/ppy/osu/issues/5578.
